packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2289,6 +2289,13 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
     return FadeTransition(opacity: _curvedAnimation!, child: child);
   }
 
+  static Widget _topLeftLayout(Widget? currentChild, List<Widget> previousChildren) {
```

Comment:
This should probably be _topStartLayout and set the Stack.alignment property to Alignment.topStart (in order to support TextDirection.rtl).